requirements.txt

```diff
@@ -2,4 +2,5 @@ numba==0.55.0
 numpy>=1.18.0,<1.22.0
 pandas>=1.3.5
 tqdm>=4.64.0
-cffi==1.15.0
\ No newline at end of file
+cffi==1.15.0
+python-dateutil==2.8.2
```

Comment:
This will no longer be necessary if start_end_years is an array. See comment in read_model_parameters file.